Content.Server/StationEvents/Components/LostPiratesSpawnRuleComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Server.StationEvents.Events;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(LostPiratesSpawnRule))]
+public sealed partial class LostPiratesSpawnRuleComponent : Component
+{
+    [DataField]
+    public string LostPiratesShuttlePath = "Maps/Shuttles/lost_pirates.yml";
+
+    [DataField]
+    public EntProtoId GameRuleProto = "Pirates";
+
+    [DataField]
+    public EntityUid? AdditionalRule;
```

Comment:
Why isn't this just using the pirates gamerule with a different prototype?

Reply:
Well... PiratesGameRule don't work at all.....